src/pages/gallary.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import gallery from 'images/gallery.png'
+import photo1 from 'images/photo-one.png'
+import photo2 from 'images/photo-two.png'
+import photo3 from 'images/photo-three.png'
+import photo4 from 'images/photo-four.png'
+import photo5 from 'images/photo-five.png'
+import photo6 from 'images/photo-six.png'
+import photo7 from 'images/photo-seven.png'
+import photo8 from 'images/photo-eight.png'
+import photo9 from 'images/photo-nine.png'
+import photo10 from 'images/photo-ten.png'
+import '../styles/gallery.css'
+
+const Gallery = () => {
+  return (
+    <>
+      <div className="banner-1">
+        <img src={gallery} className="img-fluid" alt="aboutimage" />
+      </div>
+      <div className="container mt-top">
+        <div className="grid-container mt-5">
+          <div className="item">
+            <h1 className="photo-gallery">
+              A Photo<br/> Gallery of <br />
```

Comment:
You can always create array for group of items and map them into single component when we can use same component for a list of items.
Keep it as suggestion for further cases. 
@saphalbhattarai